src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -12,17 +12,20 @@ public class StandardOutputVisitor : XmlTestExecutionVisitor
         readonly object consoleLock;
         readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
         readonly string defaultDirectory;
+        readonly bool showProgress;
 
         public StandardOutputVisitor(object consoleLock,
                                      string defaultDirectory,
                                      XElement assemblyElement,
                                      Func<bool> cancelThunk,
-                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null)
+                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null,
+                                     bool showProgress = false)
```

Comment:
Do we know if constructor is used outside this library? If so adding this new default parameter would be a binary breaking change.